app/init.php

```diff
@@ -302,6 +302,16 @@
 const METRIC_NETWORK_REQUESTS  = 'network.requests';
 const METRIC_NETWORK_INBOUND  = 'network.inbound';
 const METRIC_NETWORK_OUTBOUND  = 'network.outbound';
+const METRIC_USERS_ACTIVE = 'users.mau';
```

Comment:
This should be MAU, we might also want DAU (daily) and WAU (weekly) later on. Actually, if not too much work, let's add them now,